media-video/cheese/cheese-3.34.0.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit gnome.org vala virtualx meson
+
+DESCRIPTION="A cheesy program to take pictures and videos from your webcam"
+HOMEPAGE="https://wiki.gnome.org/Apps/Cheese"
+
+LICENSE="GPL-2+"
+SLOT="0/8" # subslot = libcheese soname version
+IUSE="gtk-doc +introspection X"
+KEYWORDS="~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+
+COMMON_DEPEND="
+	>=dev-libs/glib-2.39.90:2
+	>=x11-libs/gtk+-3.13.4:3[introspection?]
+	>=gnome-base/gnome-desktop-2.91.6:3=
+	>=media-libs/libcanberra-0.26[gtk3]
+	>=media-libs/clutter-1.13.2:1.0[introspection?]
+	>=media-libs/clutter-gtk-0.91.8:1.0
+	media-libs/clutter-gst:3.0
+	media-libs/cogl:1.0=[introspection?]
+
+	media-video/gnome-video-effects
+	x11-libs/gdk-pixbuf:2[jpeg,introspection?]
+
+	>=media-libs/gstreamer-1.4:1.0[introspection?]
+	>=media-libs/gst-plugins-base-1.4:1.0[introspection?,ogg,pango,theora,vorbis]
+
+	introspection? ( >=dev-libs/gobject-introspection-0.6.7:= )
+	X? (
+		>=media-libs/gst-plugins-base-1.4:1.0[X]
+		x11-libs/libX11
+		x11-libs/libXtst
+	)
+"
+
+BDEPEND="
+	app-text/docbook-xml-dtd:4.3
+	>=dev-util/gtk-doc-am-1.14
+	>=dev-util/intltool-0.50
+	virtual/pkgconfig
+"
+
+RDEPEND="${COMMON_DEPEND}
+	>=media-libs/gst-plugins-bad-1.4:1.0
+	>=media-libs/gst-plugins-good-1.4:1.0
+
+	>=media-plugins/gst-plugins-jpeg-1.4:1.0
+	>=media-plugins/gst-plugins-v4l2-1.4:1.0
+	>=media-plugins/gst-plugins-vpx-1.4:1.0
+"
+# libxml2+gdk-pixbuf required for glib-compile-resources
+DEPEND="${COMMON_DEPEND}
+	$(vala_depend)
+	dev-libs/appstream-glib
+	dev-libs/libxml2:2
+	dev-libs/libxslt
+	dev-util/glib-utils
+	dev-util/itstool
+	x11-base/xorg-proto
+"
+
+PATCHES=( "${FILESDIR}/${PN}-3.34.0-buildfix.patch" )
+
+src_prepare() {
+	vala_src_prepare
+
+	default
+}
+
+src_configure() {
+	local emesonargs=(
+		-Dgtk_doc=$(usex gtk-doc true false)
+		-Dintrospection=$(usex introspection true false)
+		-Dman=true
+		-Dtests=false
```

Comment:
We're disabling tests now, whereas they were enabled before. Why can't we enable tests with Meson?